### PR TITLE
Fixed the python controller so we can set controls

### PR DESCRIPTION
**Pull request by alawibaba**
Hey folks, this fixes some minor issues with the python controller. Also fixed another python test that was failing.

### CODE REVIEW

File: service/test/test_controller.py [hunk 1]
```diff
@@ -45,21 +45,24 @@ class LocalAgent(geopmdpy.runtime.Agent):
...
         return [("TIME", geopmdpy.topo.DOMAIN_BOARD, 0),
                 ("INSTRUCTIONS_RETIRED", geopmdpy.topo.DOMAIN_PACKAGE, 0)]
 
     def get_controls(self):
-        return []
+        return [("FREQUENCY", geopmdpy.topo.DOMAIN_PACKAGE, 0)]
 
     def update(self, signals):
+        if self._cpu_freq is None:
+            self._cpu_freq = geopmdpy.pio.read_signal('CPU_FREQUENCY_MAX', 'board', 0)
```
Comment (cmcantalupo) [inline service/test/test_controller.py]: put this call in __init__()?

**Review by cmcantalupo — COMMENTED** [review]
Thanks Ali!

File: service/test/test_controller.py (hunk 1 above)
Comment (alawibaba) [reply service/test/test_controller.py]: Done!

**Comment by alawibaba** [comment]
Thank you!

**Pull request merged**